spec/02-integration/05-proxy/10-handler_spec.lua

```diff
@@ -0,0 +1,154 @@
+local helpers = require "spec.helpers"
+
+describe("OpenResty handlers", function()
+
+  describe("rewrite on global plugin", function()
```

Comment:
Would have been slightly better to have the sentences be slightly more specific when tests run with `--o=gtest` flag:
```lua
describe("OpenResty phases", function()
  describe("rewrite_by_lua", function()
    describe("enabled on all APIs", function()
      it("runs")
    end)
    describe("enabled on a specific APIs", function()
      it("doesn't run")
    end)
  end)
end)
```